MOcov/@MOcovMFileCollection/write_xml_file.m

```diff
@@ -17,11 +17,11 @@ function write_xml_file(obj, output_fn)
     notify(monitor,sprintf('Writing xml files in %s', output_fn));
 
     % write header
-    overall_coverage=compute_coverage(obj);
+    [overall_coverage,lines_covered,lines_valid]=compute_coverage(obj);
     notify(monitor,sprintf('Overall coverage is %.3f', overall_coverage));
     header=sprintf(['<?xml version="1.0"?>\n'...
-                    '<coverage line-rate="%.3f" branch-rate="%.3f">'],...
-                    overall_coverage,branch_rate);
+                    '<coverage line-rate="%.3f" branch-rate="%.3f" lines-covered="%d" lines-valid="%d" branches-covered="0" branches-valid="0" complexity="0">'],...
```

Comment:
Could you please limit the line length here as done elsewhere in the code, by splitting up this line into multiple ones and use line continuation?

Reply:
Thanks!